Libraries/Lists/VirtualizedSectionList.js

```diff
@@ -345,12 +345,17 @@ class VirtualizedSectionList<
       keyEventHandler = this.props.enableSelectionOnKeyPress
         ? this._handleKeyDown
         : null;
-    } // ]TODO(macOS ISS#2323203)
+    }
+    const preferredScrollerStyleDidChangeHandler = this.props
+      .onPreferredScrollerStyleDidChange; // ]TODO(macOS ISS#2323203)
```

Comment:
nit: is the `]TODO` intentional?

Reply:
It is, because the range of lines that are a delta from upstream has changed from ending at line 348 to 350.